ios/FluentUI/Label/LabelTokenSet.swift

```diff
@@ -11,10 +11,24 @@ import UIKit
 public enum TextColorStyle: Int, CaseIterable {
     case regular
     case secondary
-    case secondaryOnColor
     case white
     case primary
     case error
+
+    func uiColor(fluentTheme: FluentTheme) -> UIColor {
```

Comment:
Doesn't it defeat the purpose of tokenization to pass in a theme in order to get the right token? Can't this be set up to just access the theme from the current window?

Reply:
That's not quite how it works. The values of `TextColorStyle` are just alternate names for a select few set of brand color tokens, and a brand color token needs a theme in order to convert it to a `UIColor`.